libs/wire-api/src/Wire/API/User/Activation.hs

```diff
@@ -168,15 +169,11 @@ modelActivationResponse = Doc.defineModel "ActivationResponse" $ do
   Doc.property "first" Doc.bool' $
     Doc.description "Whether this is the first successful activation (i.e. account activation)."
 
--- FUTUREWORK: de-deduplicate work with JSON instance for 'UserIdentity'?
```

Comment:
(This is a good example for something that we should have moved to a separate branch and merged long ago, to make for smaller PRs.)